### PR TITLE
Added Surface_vdW_to_Bidentate reaction family

### PR DESCRIPTION
**Pull request by kblondal**
Added a new catalysis reaction family:
```
If a vdW adsorbate has an internal double or a triple bond, reduce the bond order and make it bidentate.
 *1#*2                   *1==*2
   :            ---->     |  |
 ~*3~ + ~*4~~           ~*3~~*4~~
```
This is an important first step for generating bidentate adsorbates as discussed in [RMG-Py issue #1993](https://github.com/ReactionMechanismGenerator/RMG-Py/issues/1993).

### CODE REVIEW

File: input/kinetics/families/Surface_vdW_to_Bidentate/rules.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        n = 0,
+        alpha = 0.0,
+        E0 = (0, 'kcal/mol'),
+        Tmin = (200, 'K'),
+        Tmax = (3000, 'K'),
+    ),
+    rank = 0,
+    shortDesc = u"""Default""",
+    longDesc = u"""
+    These values are made up.
```
Comment (mazeau) [inline input/kinetics/families/Surface_vdW_to_Bidentate/rules.py:22]: :'(

File: input/kinetics/families/Surface_vdW_to_Bidentate/groups.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+name = "Surface_vdW_to_Bidentate/groups"
+shortDesc = u""
+longDesc = u"""
+If a vdW adsorbate has an internal double or a triple bond, reduce the bond order and make it bidentate. 
+
+ *1#*2                   *1==*2
```
[the remark this answers is not in the document]
Comment (kblondal) [reply input/kinetics/families/Surface_vdW_to_Bidentate/groups.py:9]: This is from a test run I did a while ago, showing only one of each of the following being added to the core.
vdW_to_bidentate_rxns

[the remark this answers is not in the document]
Comment (mazeau) [reply input/kinetics/families/Surface_vdW_to_Bidentate/groups.py:9]: okay, I'm happy to approve this then!

**Review by mazeau — APPROVED** [review]

**Comment by LucianaMB4920** [comment]
I did some tests with this new bidentate family using my Heptyne input file:
1)	**Production:** with RMG-Py (master) in Windows Subsystem for Linux, the last releases are rmg-3.0.0 and rmgdatabase-3.0.0. 
	-The final model core has 48 species and 47 reactions
        -The final model edge has 429 species and 945 reactions
2)	**Developer** with RMG-database (branch vdW_to_Bidentate) in a virtual machine. And I got:
        -The final model core has 54 species and 53 reactions
        -The final model edge has 436 species and 957 reactions
The output.html file of the Developer test shows these 6 reactions of the family Surface_vdW_to_Bidentate in the core:
 
image


And I add that in the core only 28 species are coincident in both tests. 
Of the 16 different species obtained with the new branch, 10 are present in the 6 new reactions of the Surface_vdW_to_Bidentate family (previous image).

**Review by LucianaMB4920 — APPROVED** [review]
I'm glad to approve!

**Comment by kblondal** [comment]
Great, thanks for reviewing! I've rebased, and this should be ready for merging if approved by a merger.

**Review by rwest — APPROVED** [review]
Don't see any problems looking at these files on github, and as people have run simulations with it, I say we merge it. And then after it's been used a while, come back and improve the rate estimates.

**Review by amarkpayne — APPROVED** [review]

**Pull request merged**